code/config/config.json

```diff
@@ -1,6 +1,6 @@
 {
   "development": {
-    "username": "isabellevillasenor",
+    "username": "rachelbuchta",
```

Comment:
I would like to leave a comment inside of the README about this needed change & need to download sequelize.